frontend/src/components/admin/Sidebar.tsx

```diff
@@ -9,7 +9,7 @@ import { classNames } from '~/utils/classNames'
 import { sidebarLinks } from '~/data/sidebarList'
 
 type Props = {
-  isOpen: Boolean
```

Comment:
why changed to type `any`?

Reply:
because it's getting error sir when I use the import createPersistedState from 'use-persisted-state'

Reply:
because of the 'use-persisted-state' package I installed
the type changed into unknow since it's located in the localstorage..

![image](https://user-images.githubusercontent.com/108642414/185317250-4fb88073-df1d-469b-a694-233c39523026.png)
![image](https://user-images.githubusercontent.com/108642414/185317507-e3632ae8-fca1-45cf-adb3-f7779b3d034d.png)